easybuild/easyconfigs/p/Python/unixccompiler-Python-2.7.11.patch

```diff
@@ -0,0 +1,11 @@
+--- EasyBuildTesting/Python-2.7.9/Lib/distutils.orig/unixccompiler.py	2014-12-10 16:59:34.000000000 +0100
```

Comment:
@gppezzi please add some comments above this line to clarify what this does, why it is need, and who came up with it (author or reference to source of this workaround)